.gitignore

```diff
@@ -0,0 +1,3 @@
+*.iso
+*.o
+iso/boot/kern
```

Comment:
Do you want to just ignore everything in iso?

Reply:
We still need the iso/boot/grub/grub.cfg file. I'll ignore everything except this file!